src/Components/Main.js

```diff
@@ -2,19 +2,20 @@ import { Component } from 'react'
 import Container from 'react-bootstrap/Container'
 import Row from 'react-bootstrap/Row'
 import Col from 'react-bootstrap/Col'
-import Image from 'react-bootstrap/Image'
+import BeastImage from './BeastImage'
+
 
 class Main extends Component {
     render() {  
         
-        const bios = this.props.HornedBeast;
+        const bios = this.props.hornedBeast;
```

Comment:
IN this component, you will use map() to iterate over hornedBeastData to render each beast.

To answer your question in Canvas: Using a loop in lab 2 could have worked for this as you can loop over the json object. However, for lab 3, map() is needed. 

Grab a TA to help 👍  

You are almost there, just have to move some pieces around here :)